src/huggingface_hub/commands/repo.py

```diff
@@ -0,0 +1,310 @@
+        except FileNotFoundError:
+            print("Looks like you do not have git installed, please install.")
+
+
+class RepoCreateCommand(BaseRepoCommand):
+    def run(self):
+        try:
+            stdout = subprocess.check_output(["git-lfs", "--version"]).decode("utf-8")
+            print(ANSI.gray(stdout.strip()))
+        except FileNotFoundError:
+            print(
+                ANSI.red(
+                    "Looks like you do not have git-lfs installed, please install."
+                    " You can install from https://git-lfs.github.com/."
+                    " Then run `git lfs install` (you only have to do this once)."
+                )
+            )
+        print("")
+
+        user = self._api.whoami(self.token)["name"]
+        namespace = self.args.organization if self.args.organization is not None else user
+
+        repo_id = f"{namespace}/{self.args.name}"
+
+        if self.args.type not in REPO_TYPES:
+            print("Invalid repo --type")
+            exit(1)
+
+        if self.args.type in REPO_TYPES_URL_PREFIXES:
+            prefixed_repo_id = REPO_TYPES_URL_PREFIXES[self.args.type] + repo_id
+        else:
+            prefixed_repo_id = repo_id
+
+        print(f"You are about to create {ANSI.bold(prefixed_repo_id)}")
+
+        if not self.args.yes:
+            choice = input("Proceed? [Y/n] ").lower()
+            if not (choice == "" or choice == "y" or choice == "yes"):
+                print("Abort")
+                exit()
+        try:
+            url = self._api.create_repo(
+                repo_id=repo_id,
+                token=self.token,
+                repo_type=self.args.type,
+                space_sdk=self.args.space_sdk,
+            )
+        except HTTPError as e:
+            print(e)
+            print(ANSI.red(e.response.text))
+            exit(1)
+        print("\nYour repo now lives at:")
+        print(f"  {ANSI.bold(url)}")
+        print("\nYou can clone it locally with the command below, and commit/push as usual.")
+        print(f"\n  git clone {url}")
+        print("")
+
+
+class RepoListCommand(BaseRepoCommand):
+    def run(self):
+        self.type = self.args.type
+        self.author = self.args.author
+        self.search = self.args.search
+        try:
+            if self.type is None or self.type == "model":
+                repos = self._api.list_models(token=self.token, author=self.author, search=self.search)
+            elif self.type == "dataset":
+                repos = self._api.list_datasets(token=self.token, author=self.author, search=self.search)
+            elif self.type == "space":
+                repos = self._api.list_spaces(token=self.token, author=self.author, search=self.search)
+        except HTTPError as e:
+            print(e)
+            print(ANSI.red(e.response.text))
+            exit(1)
+        print("\nYour repos:")
+        for repo in repos:
+            print(f"  {ANSI.bold(repo.id)}")
+        print("")
```

Comment:
```suggestion
        for repo in repos:
            print(repo.id)
```

I would go straight to the point => only print the ids. The advantage of not being verbose is that the output can be piped to another command directly.

For example:
```
huggingface-cli repo list --author=Wauplin | wc -l
```
=> counts the number of models on the Hub for author "Wauplin".

This is much harder to do when there's more colors/formatting.